### DISCUSSION TITLE
i want to use this project for comunicating between iphone and esp32 module. is this possible?

### DISCUSSION THREAD

**Discussion by digerati73** [kenjdavidson/react-native-bluetooth-classic#264] [Q&A]
i have been programming iphone app for several months.

i made it with swiftUI using bluetooth low energy .

but my client want me to develop it with bluetooth classic
because bluetooth low energy is not compatible with many phones.

so, i must re-coding from start.

i have been searching for swift ios app using bluetooth classic (br/edr) 
but  i can't find it 

i think apple doesn't support bluetooth classic well.
of course, apple support bluetooth low energy very well.

finally, i can't find sample code , swift ios app using bluetooth classic
but
i find your project made by react-native.

when i first make a plan for this project , 
i want to make it with react-native.

and 
after some weeks later 
i make a program with swift by xcode.

and 
now 
i come back to your project.

could this project be used for communicating between iphone and esp32 module  with bluetooth classic ?

**Comment by kenjdavidson** [comment]
No this won't work. Not because of the library, but because the ESP32 is not MFI certified and not supported by apple Core Bluetooth.

**Reply by digerati73** [reply]
oh my god.... thank you for your kind reply
